tests/ModelHelperTest.php

```diff
@@ -3,11 +3,14 @@
 namespace AlgoliaSearch\Tests;
 
 use AlgoliaSearch\Tests\Models\Model1;
+use AlgoliaSearch\Tests\Models\Model11;
```

Comment:
Class `AlgoliaSearch\Tests\Models\Model11` does not exists. The line should be removed.